src/views/PeopleSearch/components/SearchFieldList/index.tsx

```diff
@@ -245,19 +263,61 @@ const SearchFieldList = ({ onSearch }: Props) => {
     return () => window.removeEventListener('popstate', onNavigate);
   }, [location.search, initialSearchParams]);
 
-  const handleUpdate = (event: ChangeEvent<HTMLInputElement>) =>
+  const handleUpdate = (event: ChangeEvent<HTMLInputElement>) => {
+    if (event.target.name === 'graduation_year') {
+      setSearchParams((sp) => ({
+        ...sp,
+        ...{ class_standing: '' },
+      }));
+    } else if (event.target.name === 'class_standing') {
+      setSearchParams((sp) => ({
+        ...sp,
+        ...{ initial_year: '', final_year: '' },
+        ...{ graduation_year: '' },
+      }));
+    }
+
     setSearchParams((sp) => ({
       ...sp,
       [event.target.name]:
         event.target.type === 'checkbox' ? event.target.checked : event.target.value,
     }));
+  };
 
   const handleEnterKeyPress = (event: KeyboardEvent<HTMLDivElement>) => {
     if (event.key === 'Enter') {
       search();
     }
   };
 
+  const handleSliderChange = (event: Event, newValue: number | number[]) => {
+    setGraduationYearRange(newValue as number[]);
+    var values = graduationYearRange.toString().split(',');
+    setSearchParams((sp) => ({
+      ...sp,
+      ...{ initial_year: values[0], final_year: values[1] },
+    }));
+    setSearchParams((sp) => ({
+      ...sp,
+      ...{ class_standing: '' },
+    }));
+  };
+
+  const handleSwitchChange = () => {
+    if (switchYearRange === true) {
+      setSearchParams((sp) => ({
+        ...sp,
+        ...{ initial_year: '', final_year: '' },
+      }));
+    } else {
+      setSearchParams((sp) => ({
+        ...sp,
+        ...{ initial_year: '', final_year: '' },
+      }));
+    }
```

Comment:
Is there a reason the if/else block has the exact same block of code?

Reply:
No reason just mistake, just removed after testing it.